records_mover/db/bigquery/loader.py

```diff
@@ -102,15 +105,21 @@ def load(self,
              schema: str,
              table: str,
              load_plan: RecordsLoadPlan,
-             directory: RecordsDirectory) -> int:
+             directory: RecordsDirectory) -> Optional[int]:
```

Comment:
We're now calling into a generic interface (`. load_from_records_directory_via_fileobj()` below which doesn't know if the implementation can actually return the number of rows loaded.  Rest assured that this one can (not that anything depends on that - this is bubbled up as Optional throughout).